packages/ckeditor5-paste-from-office/docs/features/paste-from-word.md

```diff
@@ -36,6 +36,13 @@ With CKEditor 5 you do not need to worry about pasting messy content from Micros
 
 This means that if you did not enable, for instance, {@link features/font font family and font size} features, this sort of formatting will be automatically stripped off when you paste content from Microsoft Word and other sources (e.g. other websites).
 
+## Related features
+
+CKEditor 5 supports a wider range of paste features, including:
+* {@link features/paste-from-google-docs Paste from Google Docs} &ndash; Allows you to paste content from Google Docs maintaining the original formatting and structure.
+* {@link features/paste-plaintext Paste plain text} &ndash; Detects when a plain text is pasted and acts accordingly.
+<!-- * {@link features/html-embed Raw HTML embedding} &ndash; Lets you paste HTML source into the edited content and keep the output. -->
```

Comment:
This here still causes the build to fail and is also not a subject of this PR/issue.

Reply:
These should've been removed, they are not present in the local copy.

Reply:
Looks like a commit was lost and not pushed yesterday. Either way, it's all good now.